cmd/collector/app/builder_flags.go

```diff
@@ -96,6 +110,14 @@ func AddFlags(flags *flag.FlagSet) {
 func AddOTELJaegerFlags(flags *flag.FlagSet) {
 	flags.String(CollectorHTTPHostPort, ports.PortToHostPort(ports.CollectorHTTP), "The host:port (e.g. 127.0.0.1:14268 or :14268) of the collector's HTTP server")
 	flags.String(CollectorGRPCHostPort, ports.PortToHostPort(ports.CollectorGRPC), "The host:port (e.g. 127.0.0.1:14250 or :14250) of the collector's GRPC server")
+
+	// auth-related flags
+	flags.String(authOIDCIssuerURL, "", "the OpenID Connect server to validate the incoming auth tokens")
```

Comment:
won't these flags start showing up on the Jaeger collector but be unsupported?

Reply:
They shouldn't, as they are part of `AddOTELJaegerFlags`.

Reply:
`AddOtelJaegerFlags` is called from the normal `AddFlags` here:

https://github.com/jaegertracing/jaeger/blob/master/cmd/collector/app/builder_flags.go#L92

Reply:
You are absolutely right:

```
$ go run ./cmd/collector/ --help | grep oidc
2020/10/23 22:13:16 maxprocs: Leaving GOMAXPROCS=12: CPU quota undefined
      --collector.auth.oidc.client-id string            this Jaeger's client ID
      --collector.auth.oidc.groups-claim string         the claim containing the group membership, where each group is considered a tenant. When absent, multi-tenancy is disabled.
      --collector.auth.oidc.issuer-ca-path string       the path to the issuer's CA
      --collector.auth.oidc.issuer-url string           the OpenID Connect server to validate the incoming auth tokens
      --collector.auth.oidc.username-claim string       the claim in the token that should be used for the username instead of the default 'sub' field
```

I should probably suppress the flags from the current collector.